src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -10,100 +15,151 @@ const StemDeniedDetails = ({
 
   return (
     <>
-      <h1>Your Edith Nourse Rogers STEM Scholarship Application</h1>
+      <h1>Your Edith Nourse Rogers STEM Scholarship application</h1>
       <div className="vads-u-background-color--primary-alt-lightest vads-u-padding--2 vads-u-margin-bottom--3">
         <h2 className="claims-alert-header vads-u-font-size--h4">
           Your application was denied on {date}.
         </h2>
       </div>
-      <h2 className="vads-u-font-family--sans vads-u-margin-y--0">
-        Why we denied your claim
-      </h2>
-      <p className="vads-u-margin--0">
+      <p className="va-introtext">
         You must meet all 3 of the eligibility criteria to be considered for the
         Rogers STEM Scholarship.
       </p>
       <h3 className="vads-u-font-family--sans vads-u-margin-bottom--0">
-        You did not meet these criteria for the Rogers STEM Scholarship:
+        You didn't meet the following criteria for the Rogers STEM Scholarship:
       </h3>
       <ul className="stem-ad-list">
-        <li className="stem-ad-list-item">
-          You don’t meet the benefit requirements for the Rogers STEM
-          Scholarship
-          <ul className="stem-ad-list-secondary">
-            <li className="stem-ad-list-item">
-              By law, all Rogers STEM Scholarship recipients must have 6 months
-              or less of Post 9/11 GI Bill (Chapter 33) benefits left (38 U.S.
-              Code § 3320).
-            </li>
+        {remainingEntitlement > 180 && (
```

Comment:
Conditional is not needed here. This will always be the reason for denial for now.